debugger/src/dap_types.rs

```diff
@@ -8,6 +8,25 @@ use std::convert::TryFrom;
 
 schemafy!(root: debugserver_types "src/debugProtocol.json");
 
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize)]
+pub struct RttChannelEventBody {
+    pub channel_number: usize,
+    pub channel_name: String,
+    pub data_format: crate::rtt::DataFormat,
+}
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize)]
+pub struct RttDataEventBody {
+    pub channel_number: usize,
+    #[doc = " RTT output"]
+    pub data: String,
+}
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize)]
+pub struct ShowMessageEventBody {
+    /// The `severity` field can be one of "information", "warning", or "error"
+    pub severity: String,
```

Comment:
How about we make this a proper enum, which then gets serialized into those three values? :) Like

```rust
#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
enum Severity {
    Information,
    Warning,
    Error
}
```

one could also put a `#[serde(rename_all = "lowercase")]` to make it all lowercase :)

Reply:
Consider it done :)